apps/etl-func/local.settings.json.example

```diff
@@ -16,6 +16,9 @@
     "COSMOS__accountEndpoint": "https://io-d-itn-common-cosno-01.documents.azure.com:443",
     "COSMOS_DBNAME": "io-messages",
     "COSMOS_MESSAGES_CONTAINER_NAME": "messages",
+    "IOCOM_COSMOS__accountEndpoint": "https://io-d-itn-common-cosno-01.documents.azure.com:443",
```

Comment:
nitpick: i would rename `COSMOS_` in `COMMON_COSMOS_` to mark that `IOCOM_COSMOS` will be the default

Reply:
[1f351ac](https://github.com/pagopa/io-messages/pull/85/commits/1f351ac5427632f95fb55298a7ca3ec76461c41d)